### PR TITLE
Update skycli config format

### LINKED ISSUE #185: Update skycli config format
- [ ] Add `version`
- [ ] Change hash to list

### PR DESCRIPTION
**Pull request by louischan-oursky**
fix #185 
depend on https://github.com/SkygearIO/skycli/pull/188

The version field is named `version` instead of `api_version`. I'm not sure if it is a good choice.

### CODE REVIEW

File: src/config.test.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+      login_id: user@example.com
+      claims:
+        email: user@example.com
+    access_token: access_token
+    refresh_token: refresh_token
+    session_id: session_id
+`;
+
+const expected = `
+version: v1
```
Comment (kiootic) [inline src/config.test.ts]: I think for consistency using `api_version` is better.

File: src/config.ts [hunk 2]
```diff
@@ -95,16 +117,119 @@ export function set(
...
+  }
+  return skycliConfig;
+}
+
 export function loadConfig() {
-  let globalConfig = load(ConfigDomain.GlobalDomain) as GlobalConfig;
-  if (!Object.keys(globalConfig).length) {
-    globalConfig = createGlobalConfig();
+  let skycliConfig = migrateSkycliConfig(load("global"));
+  if (!Object.keys(skycliConfig).length) {
```
Comment (kiootic) [inline src/config.ts]: We do not save the migrated configuration?

File: src/config.test.ts (hunk 1 above)
Comment (louischan-oursky) [reply src/config.test.ts]: So the field name is the same but it is entirely different thing than the API version we have in Controller. Is it OK? @carmenlau

Comment (carmenlau) [reply src/config.test.ts]: I think it is okay to use name `api_version` with different value, skycli config should have its own version.

**Comment by louischan-oursky** [comment]
@kiootic comment addressed.

**Review by kiootic — APPROVED** [review]

**Pull request merged**